### ISSUE TITLE
Paginate two or more collections in one action

### ISSUE THREAD

**Issue by cseelus** [ddnexus/pagy#20]
If I understand it correctly, currently the only way to rename the `page` param, is to redefine `pagy_get_params`.

Will paginate for example allows to do something like this:

```ruby
paginate @my_collection, param_name: 'custom_param_name'
```

Which in turn easily enables to have more than one collection paginated per action.

@ddnexus, would something like this be a feature that you are willing to add (yourself or via a PR) or is this outside the scope of Pagy?

**Comment by ddnexus** [comment]
You can do it right now by doing:
```ruby
@pagy, @records = pagy(@mycollection, page: param[:custom_param_name])
```

**Comment by ddnexus** [comment]
and BTW... `pagy_get_params` is the way to massage the params. (see [Customizing the params](https://ddnexus.github.io/pagy/how-to#customizing-the-params)). 

The proper way to change the param name (but for all instances) would be overriding `pagy_get_vars`, which would work as a default for all the `pagy` calls, but... just pass a variable to the pagy method and it will override the default.

**Comment by cseelus** [comment]
Yes, I also thought/wrote `pagy_get_params` was the way to go to modify the page links, but to paginate two collections in one action/on one page, you would need to define two helpers that both would redefine `pagy_get_params`?

**Comment by ddnexus** [comment]
> but to paginate two collections in one action/on one page, you would need to define two helpers that both would redefine pagy_get_params?

Absolutely not:

```ruby
@pagy, @records = pagy(@mycollection)               # uses params[:page]
@custom_pagy1, @custom_records = pagy(@mycollection1, page: param[:custom_param_name1])
@custom_pagy2, @custom_records2 = pagy(@mycollection2, page: param[:custom_param_name2])
```

**Comment by cseelus** [comment]
`<%= pagy_nav @custom_pagy1 %>` will generate links like `<a href="/collection?page=2">` though, using `page` as the page param instead of `custom_param_name1`.

**Comment by ddnexus** [comment]
True! And wrong! I have to fix that!
